ci-operator/config/codeready-toolchain/workload-analyzer/codeready-toolchain-workload-analyzer-master.yaml

```
@@ -0,0 +1,56 @@
+base_images:
+  base:
+    name: "4.17"
+    namespace: ocp
+    tag: base
+build_root:
+  project_image:
+    dockerfile_path: openshift-ci/Dockerfile.tools
+images:
+- dockerfile_path: test/e2e/alertmanager-mock/build/Dockerfile
+  from: base
+  to: alertmanager-mock
+- dockerfile_path: openshift-ci/build/Dockerfile.openshift-ci
+  from: base
+  to: workload-analyzer
+releases:
+  initial:
+    integration:
+      name: "4.17"
+      namespace: ocp
+  latest:
+    integration:
+      include_built_images: true
+      name: "4.17"
+      namespace: ocp
+resources:
+  '*':
+    requests:
+      cpu: "3"
+      memory: 250Mi
+tests:
+- as: e2e
+  steps:
+    cluster_profile: devsandboxci-aws
+    env:
+      BASE_DOMAIN: devsandboxci.devcluster.openshift.com
+    test:
+    - as: test
+      cli: latest
+      commands: |
+        export WEBHOOK_URL=$(cat /usr/local/sandbox/ci-secrets/webhook_url)
+        make test-e2e-in-ci
+      credentials:
+      - mount_path: /usr/local/sandbox/ci-secrets
+        name: ci-secrets
+        namespace: test-credentials
```

you can remove the ReportPortal stuff
```suggestion
      commands: make test-e2e-in-ci
```